internal/drivers/ankabuild/driver.go

```diff
@@ -78,24 +78,36 @@ func (c *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (in
 	}
 	var id = response.Body[0]
 	vm := &vmResponse{}
-	for i := 1; i <= 50000; i++ {
+	for i := 1; i <= 100; i++ {
```

Comment:
there are other ways to achieve this with a context, i used something similar to what brad used for ssh connections here https://github.com/drone-runners/drone-runner-aws/pull/6/files or here https://golang.cafe/blog/golang-context-with-timeout-example.html
it removes the reliance on sleeps. 
i do understand the urgency of this though, but i can see the sleeps / timing being an issue again if there are networking issues.

Reply:
Agreed - I think for now we should come back to making this change.